applications/schedview-snapshot/templates/vault-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+# Follow the example of plot-navigator
+
+apiVersion: ricoberger.de/v1alpha1
+kind: VaultSecret
+metadata:
+  name: {{ template "schedview-snapshot.fullname" . }}
+  labels:
+    {{- include "schedview-snapshot.labels" . | nindent 4 }}
+spec:
+  path: "{{ .Values.global.vaultSecretsPath }}/butler-secret"
```

Comment:
The butler-secret pattern is no longer supported and we're trying to move away from it. You'll see that plot-navigator has an if conditional on separateSecrets. Since this is a new secret, you should use only the separateSecrets approach, which is the new way to do this.

In other words, path should instead be:
```
path: "{{ .Vault.global.vaultSecretsPath }}/schedview-snapshot"
```

Reply:
How do I create (or who do I ask to create) an entry in the vault for schedview-snapshot? How do I create (or who do I ask to create) the secrets that need to go into the vault?

Reply:
You can ask on #ops-usdf on Slack for help with Vault. I'm not sure if they'll want to give you access so that you can make the change yourself or make it for you. Where you get the secret depends on the answer to the above question: either someone should give you a new secret, or you (or someone helping you) will get it from the nublado Vault secret.